.gitignore

```diff
@@ -1 +1,5 @@
-.idea/
\ No newline at end of file
+.idea/
+out/
+
+Main.java
```

Comment:
Remove `Main.java` from gitignore.

Reply:
At this moment is better to maintain it, because is empty.doesn't make sense to push it